test/unit/app/presenters/publishing_api/editionable_worldwide_organisation_presenter_test.rb

```diff
@@ -183,4 +183,28 @@ def present(...)
 
     assert_equal [lead_organisation_2.content_id, lead_organisation_1.content_id], presented_item.content.dig(:links, :sponsoring_organisations)
   end
+
+  test "does not convert an absent body to govspeak" do
+    worldwide_org = create(:editionable_worldwide_organisation)
+
+    # rubocop:disable Rails/SkipsModelValidations
```

Comment:
What's the reason for these Rubocop skips? Just curious more than anything

Reply:
I've explained the reason in the commit message.

Reply:
Whoops, didn't see that, all good then